templates/post-multiple-authors.adoc

```diff
@@ -36,6 +40,16 @@ AUTHOR_NAME <https://github.com/GITHUB_USERNAME>
 // Replace AUTHOR_NAME with your name as you'd like it to be displayed, eg: Laura Cowen
 //
 // Example post: 2020-02-12-faster-startup-Java-applications-criu.adoc
+//
+// If adding image into the post add :
+// -------------------------
+// [.img_border_light]
+// image::img/blog/FILE_NAME[IMAGE CAPTION ,width=70%,align="center"]
+// -------------------------
+// "[.img_border_light]" = This adds a faint grey border around the image to make its edges sharper. Use it around screenshots but not           
```

Comment:
There is also a `[.img_border_dark]` (I think that's right - check the exact name with Ellen in the website channel) class you can use for really dark images (most images that need the edges sharpening are screenshots taken against a light background, so the light grey border is best. But where it's a darker image, you might find that the darker grey border works better.

Can you add mention of that in each template too?

Reply:
Done, waiting on confirmation from Ellen

Reply:
I found the original issue that contained the info: https://github.com/OpenLiberty/openliberty.io/issues/1496

Reply:
Confirmed and Done